source/_data/plugins.yml

```diff
@@ -70,7 +70,7 @@
       keywords: [testing-library, capybara]
     
     - name: PickleJS
-      description: An addition to the Cucumber plugin, featuring a collection of phrases you can use for common actions (ex: "I click on an <Element>", "I should see an <Element>")
+      description: 'An addition to the Cucumber plugin, featuring a collection of phrases you can use for common actions (ex: "I click on an <Element>", "I should see an <Element>")'
```

Comment:
The presence of `ex: ` in the `description` was b0rking the build. I've explicitly quoted the string to address this.